app/src/main/java/com/powerhungers/fomezero/feature/standbyClass/ConsumerFragment.kt

```diff
@@ -1,24 +1,49 @@
 package com.powerhungers.fomezero.feature.standbyClass
 
+import android.content.Intent
+import android.net.Uri
 import android.os.Bundle
-import androidx.fragment.app.Fragment
 import android.view.LayoutInflater
 import android.view.View
 import android.view.ViewGroup
-import com.powerhungers.fomezero.R
+import androidx.fragment.app.Fragment
+import androidx.navigation.fragment.findNavController
+import com.powerhungers.fomezero.databinding.FragmentConsumerBinding
 
 class ConsumerFragment : Fragment() {
```

Comment:
```suggestion
class ConsumerHomeFragment : Fragment() {
```
o que acha de alterar esse nome pra ficar mais claro do que essa classe se trata?